### ISSUE TITLE
[Bug]: mangrove net change widget: graph displays annual change instead of cumulative change from start-year to end-year

### ISSUE THREAD

**Issue by lhilarides** [Vizzuality/mangrove-atlas#533]
![image](https://user-images.githubusercontent.com/3919721/189946965-f43cf8a5-750b-4db7-8531-7abc632b12f4.png)

I finally figured out what's wrong with the numbers in the mangrove net change widget: the text mentions the cumulative change (which is good!). but the graph displays the differences from year to year. The graph should also show the cumulative net change, so that in this screenshot the line starts on the left at 0 and ends on the right at -161,90 km2 in 2020

**Comment by lhilarides** [comment]
Small addition: can you rephrase the word "net result" to "net change" ?

**Comment by dhakelila** [comment]
This is done and the new calculations are ready in staging for your approval. 
Let's follow the conversation at: https://github.com/Vizzuality/mangrove-atlas/issues/421
